src/js/utils/strings.ts

```diff
@@ -94,25 +99,32 @@ export function seconds(str, frameRate) {
 }
 
 // Convert an offset string to a number; supports conversion of percentage offsets
-export function offsetToSeconds(offset, duration, frameRate) {
-    if (isString(offset) && offset.slice(-1) === '%') {
+export function offsetToSeconds(offset: string): number;
+export function offsetToSeconds(offset: string, duration: number): number;
+export function offsetToSeconds(offset: string, duration: number, frameRate: number): number;
```

Comment:
It looks like some methods like 
> offsetToSeconds(offset, duration, frameRate)

aren't used outside of tests. Not sure if we should keep them.